test/JDBC/input/BABEL_4320.sql

```diff
@@ -0,0 +1,69 @@
+-- dummy test to check if schema is being rewritten
+IF (EXISTS (select * from information_schema.test))
```

Comment:
Other rewrites like truncate identifiers were also not working, I think it would be great to test some other rewrite along with isc_tsql

Reply:
Done